### PR TITLE
Make plural after a number lower case

### PR DESCRIPTION
**Pull request by robwebset**
Pull request to support adding the 's' as lower case if ending in a numeric value.

Solves request #77 

Thanks
Rob

### CODE REVIEW

**Comment by robwebset** [comment]
OK, we should be good now - first error was for spaces/tabs , the second was me adding a test into the test.js

Thanks
Rob

**Comment by blakeembrey** [comment]
How about just inverting the current checks to do `word.toLowerCase() === word` to check it "did not" upper-case? That would skip the need for checking numbers and other related cases.

**Comment by robwebset** [comment]
I tried that, unfortunately it doesn't work with things like "Work Order2" and "SoundFX2"

**Comment by robwebset** [comment]
Sorry, I'm an idiot! It does work - Corrected it now :)

**Comment by robwebset** [comment]
OK, sorted everything out now ... hopefully!

File: pluralize.js [hunk 1]
```diff
@@ -49,7 +49,10 @@
     // Tokens are an exact match.
     if (word === token) return token;
 
-    // Upper cased words. E.g. "HELLO".
+    // Lower cased words. E.g. "hello".
+    if (word === word.toLowerCase()) return token.toLowerCase();
```
Comment (blakeembrey) [inline pluralize.js]: Looks good. Is it possible to remove the title case check now though and fallback to it by default if neither of these cases match? No need to check if that should be the default now anyhow.

Comment (robwebset) [reply pluralize.js]: I think if we did that things like soundFX, oDonald etc would fail

**Comment by wernerm** [comment]
Why is this not being merged?

**Pull request merged**